tcms/testplans/static/testplans/js/get.js

```diff
@@ -0,0 +1,145 @@
+            }
+        } else {
+            testCaseListHtml = `<div class="list-group-item">
+                                                <div class="list-group-item-header">
+                                                    <div class="list-view-pf-main-info">No testcases</div>
+                                                </div>
+                                            </div>`;
+        }
+        mainTestCaseContainer.innerHTML = testCaseListHtml;
+    };
+
+    var generateConfirmedCaseItem = function(testCase) {
+        return `<div class="list-group-item">
+                    <div class="list-group-item-header">
+                        <div class="list-view-pf-expand">
+                            <span class="fa fa-angle-right"></span>
+                        </div>
+                        <div class="list-view-pf-checkbox">
+                            <input type="checkbox"></div>
+                        <div class="list-view-pf-actions">
+                            <div class="dropdown pull-right dropdown-kebab-pf">
+                                <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
+                                        data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                                    <span class="fa fa-ellipsis-v"></span>
+                                </button>
+                                <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
+                                    <li><a href="#">Action</a></li>
+                                    <li><a href="#">Another Action</a></li>
+                                    <li><a href="#">Something Else Here</a></li>
+                                    <li role="separator" class="divider"></li>
+                                    <li><a href="#">Separated Link</a></li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="list-view-pf-main-info">TC-${testCase.id}: ${testCase.summary}</div>
+                    </div>
+                    <div class="list-group-item-container container-fluid hidden">
+                        <div class="close">
+                            <span class="pficon pficon-close"></span>
+                        </div>
+                        <div class="row">
+                            <div class="col-md-12">Some test case widgets</div>
+                        </div>
+                    </div>
+                </div>`;
+    };
+
+    var addTestCasesAction = function() {
+        $("#confirmed-testcases input[type='checkbox']").change(function(e) {
+            if ($(this).is(":checked")) {
+                $(this).closest('.list-group-item').addClass("active");
+            } else {
+                $(this).closest('.list-group-item').removeClass("active");
+            }
+        });
```

Comment:
The `active` class is a visual aid only from the legacy page. It turns the current row pale yellow from what I can tell. I don't think this is used for anything else so let's remove it until it becomes necessary.